### PR TITLE
Only output demurrage form from to_human()

### PR DESCRIPTION
**Pull request by sublimator**
Reopening for discussion, as @shekenahglory is still having issues with this

### CODE REVIEW

**Comment by sublimator** [comment]
#492

**Comment by shekenahglory** [comment]
This and the tecOVERSIZE addition are preventing me from properly importing and validating certain ledgers - I'd like to get a new release candidate with these fixes

File: src/core/amount.js [hunk 1]
```diff
@@ -941,7 +941,7 @@ Amount.prototype.to_text_full = function(opts) {
   }
   return this._is_native
       ? this.to_human() + '/XRP'
-      : this.to_text() + '/' + this._currency.to_json()
+      : this.to_text() + '/' + this._currency.to_human()
```
Comment (clark800) [inline src/core/amount.js]: What is this change for?

File: src/core/currency.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var _ = require('extend');
```
Comment (wltsmrz) [inline src/core/currency.js]: Did you mean to require lodash?

**Comment by clark800** [comment]
Closing per @sublimator's request, he will make a JIRA task instead

**Pull request closed without merging**